src/api-reference/expense/expense-report/company-card-transaction-resource.markdown

```diff
@@ -3,10 +3,6 @@ title: Company Card Transactions
 layout: reference
 ---
 
```

Comment:
No TOC in this file?

Reply:
Because there is only one call with no request parameters and a singular request schema, there isn't too much value added by including a TOC, as it's not going to help much with navigation. Should we include a TOC just for consistency sake? I flip back and forth on the idea.